### PR TITLE
docs(sem): add secret migration doc

### PR DESCRIPTION
**Pull request by nerda-codes**

### CODE REVIEW

File: identity-and-access-management/secret-manager/api-cli/migrate-secrets-in-folders.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  paragraph: Learn how to migrate your secrets into your folders using the CLI and Go
+tags: secret-management folders migration secret-manager go sdk cli
+categories:
+  - identity-and-access-management
+dates:
+  validation: 2023-09-25
+  posted: 2023-09-25
+---
+
+This documentation shows you how to migrate your secrets into [folders](/identity-and-access-management/secret-manager/concepts/#folder) using the [Scaleway CLI](https://github.com/scaleway/scaleway-cli#installation) and Go. we have created a GitHub repository that contains the code you need to create a folder and to migrate your secrets into this folder using your secrets' IDs.
```
Comment (bene2k1) [inline identity-and-access-management/secret-manager/api-cli/migrate-secrets-in-folders.mdx:16]:
```suggestion
This documentation shows you how to migrate your secrets into [folders](/identity-and-access-management/secret-manager/concepts/#folder) using the [Scaleway CLI](https://github.com/scaleway/scaleway-cli#installation) and the Go programming language. we have created a GitHub repository that contains the code you need to create a folder and to migrate your secrets into this folder using your secrets' IDs.
```

I'll let you decide for this one, but maybe not everyone knows what Go is.

File: identity-and-access-management/secret-manager/api-cli/migrate-secrets-in-folders.mdx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    export SCW_DEFAULT_PROJECT_ID=<Scaleway default Project>
+    export SCW_DEFAULT_REGION="fr-par"
+    export SCW_API_URL="https://api.scaleway.com"
+    ```
+
+3. Use the following command to access the `tutorial-sem-migration` directory:
+    ```
+    cd tutorial-sem-migration
+    ```
+4. Open a Text editor of your choice and open the `main.go` file inside the `tutorial-sem-migration` directory.
```
Comment (bene2k1) [inline identity-and-access-management/secret-manager/api-cli/migrate-secrets-in-folders.mdx:47]:
```suggestion
4. Open a text editor of your choice and open the `main.go` file inside the `tutorial-sem-migration` directory.
```

**Review by bene2k1 — APPROVED** [review]

**Review by jsudraud — APPROVED** [review]

File: identity-and-access-management/secret-manager/api-cli/migrate-secrets-in-folders.mdx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  paragraph: Learn how to migrate your secrets into your folders using the CLI and Go
+tags: secret-management folders migration secret-manager go sdk cli
+categories:
+  - identity-and-access-management
+dates:
+  validation: 2023-09-25
+  posted: 2023-09-25
+---
+
+This documentation shows you how to migrate your secrets into [folders](/identity-and-access-management/secret-manager/concepts/#folder) using the [Scaleway CLI](https://github.com/scaleway/scaleway-cli#installation) and the Go programming language. we have created a GitHub repository that contains the code you need to create a folder and to migrate your secrets into this folder using your secrets' IDs.
```
Comment (jcirinosclwy) [inline identity-and-access-management/secret-manager/api-cli/migrate-secrets-in-folders.mdx:16]:
```suggestion
This documentation shows you how to migrate your secrets into [folders](/identity-and-access-management/secret-manager/concepts/#folder) using the [Scaleway CLI](https://github.com/scaleway/scaleway-cli#installation) and the Go programming language. We have created a GitHub repository containing the necessary code to create the folder to which you will migrate your secrets using their IDs.
```

Comment (jcirinosclwy) [reply identity-and-access-management/secret-manager/api-cli/migrate-secrets-in-folders.mdx:16]: Just a suggestion. Feel free to ignore and just Add the capital W in we :)

File: identity-and-access-management/secret-manager/api-cli/migrate-secrets-in-folders.mdx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    ```
+
+3. Use the following command to access the `tutorial-sem-migration` directory:
+    ```
+    cd tutorial-sem-migration
+    ```
+4. Open a text editor of your choice and open the `main.go` file inside the `tutorial-sem-migration` directory.
+5. Replace `my-folder`, lines `24`, `27`, and `34` with the name of your folder.
+    <Message type="important">
+     Your folder name can only contain alphanumeric characters, dots, underscores and dashes.
```
Comment (jcirinosclwy) [inline identity-and-access-management/secret-manager/api-cli/migrate-secrets-in-folders.mdx:50]:
```suggestion
     Your folder name can only contain alphanumeric characters, dots, underscores, and dashes.
```

**Review by jcirinosclwy — APPROVED** [review]

**Pull request merged**